.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,67 @@
+name: Bug Report
+description: Something isn't working as expected
+labels: [bug]
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thank you for contributing to Swift-DocC! 
+      
+      Before you submit your issue, please complete each text area below with the relevant details for your bug, and complete the steps in the checklist
+- type: textarea
+  attributes:
+    label: Description
+    description: |
+      A short description of the incorrect behavior. 
+
+      If you think this issue has been recently introduced and did not occur in an earlier version, please note that. If possible, include the last version that the behavior was correct in addition to your current version.
+  validations:
+    required: true
+- type: checkboxes
+  attributes:
+    label: Checklist
+    options:
+    - label: If possible, I've reproduced the issue using the `main` branch of this package.
+      required: false
+    - label: This issue hasn't been addressed in an [existing GitHub issue](https://github.com/apple/swift-docc/issues).
+      required: true
+- type: textarea
+  attributes:
+    label: Expected Behavior
+    description: Describe what you expected to happen.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Actual behavior
+    description: Describe or copy/paste the behavior you observe.
+  validations:
+    required: false
+- type: textarea
+  attributes:
+    label: Steps To Reproduce
+    description: |
+      Explanation of how to reproduce the incorrect behavior. 
+      
+      This could include a reduced version of your documentation bundle, or a link to the documentation (or code) that is exhibiting the issue.
+    placeholder: |
+      1. In this environment...
+
+      2. With this config...
+      
+      3. Run '...'
+      
+      4. See error...
+  validations:
+    required: false
+- type: input
+  attributes:
+    label: Swift-DocC Version Information
+    description: The version of Swift-DocC used to reproduce this issue.
+    placeholder: "'5.6.0' for example, or a commit hash"
```

Comment:
Review notes for @ethan-kusters 
1. The render key is not supported on type: input
2. The placeholder value: `'5.6.0' for example, or a commit hash` must be quoted for some reason. Otherwise, we'll get a yaml parse error.